src/main/java/koreatech/in/service/admin/AdminUserServiceImpl.java

```diff
@@ -142,6 +140,13 @@ public User getUserForAdmin(int id) {
         return user;
     }
 
+    @Override
+    public StudentResponse getStudentForAdmin(Integer userId) {
+        User user = adminUserMapper.getUserById(userId);
+        Optional.ofNullable(user).orElseThrow(() -> new BaseException(INQUIRED_USER_NOT_FOUND));
```

Comment:
한번에 처리하기 위해 이렇게 코드를 작성할 수도 있을 것 같아요!
참고만 해도 괜찮을 것 같아요.

```suggestion
        User user = Optional.ofNullable(adminUserMapper.getUserById(userId))
                            .orElseThrow(() -> new BaseException(INQUIRED_USER_NOT_FOUND));
```

Reply:
수정하겠습니다!